griptape/chunkers/base_chunker.py

```diff
@@ -31,7 +31,7 @@ def chunk(self, text: TextArtifact | str) -> list[TextArtifact]:
 
         return [TextArtifact(c) for c in self._chunk_recursively(text)]
 
-    def _chunk_recursively(self, chunk: str, current_separator: Optional[ChunkSeparator] = None) -> list[str]:
+    def _chunk_recursively(self, chunk: str, current_separator: Optional[ChunkSeparator] = None) -> list[str]:  # noqa: C901
```

Comment:
I'm not sure I like adding a rule that only leads to one violation being "fixed", while the other 3 violations are immediately ignored.

Is the goal to just get authors of PRs to think twice before adding complex functions? (but still allow them if the complexity is warranted?)

Reply:
> Is the goal to just get authors of PRs to think twice before adding complex functions?

Yes. Ignoring these is 100% lazyness on my end -- there was not an immediately easy refactor available. I'm more concerned about new code, but I can definitely spend the time fixing these now.